files/shunting-yard-config.yml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
I don't see shunting-yard-config.yml file being copied during build
can you remove shunting-yard-config.yml and shunting-yard-variables.conf from repo?

Reply:
Removed shunting-yard-config.yml.